src/containers/GroupAdmin/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import Banner from 'components/layout/Banner';
+import Paper from 'components/layout/Paper';
+import { Skeleton } from '@material-ui/lab';
+import Navigation from 'components/navigation/Navigation';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import EditIcon from '@material-ui/icons/Edit';
+import AddIcon from '@material-ui/icons/Add';
+import Http404 from 'containers/Http404';
+import { useGroup } from 'api/hooks/Group';
+import { useMemberships } from 'api/hooks/Membership';
+import { useState } from 'react';
+import UpdateGroupModal from 'containers/GroupAdmin/components/UpdateGroupModal';
+import MemberListItem from 'containers/GroupAdmin/components/MemberListItem';
+import AddMemberModal from './components/AddMemberModal';
+import MembersCard from 'containers/Pages/specials/Index/MembersCard';
+
+const useStyles = makeStyles((theme) => ({
+  gutterBottom: {
+    marginBottom: theme.spacing(2),
+  },
+  button: {
+    textTransform: 'none',
+  },
+}));
+
+const Group = () => {
+  const classes = useStyles();
+  const params = useParams();
+
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [addMemberModalIsOpen, setAddMemberModalIsOpen] = useState(false);
+  const handleClose = () => {
+    setModalIsOpen(false);
+  };
+
+  const addMemberHandleClose = () => {
+    setAddMemberModalIsOpen(false);
+  };
+  const slug = params.slug.toLowerCase();
+  const { data, isLoading: isLoadingGroups, isError } = useGroup(slug);
+
+  const hasWriteAcccess = data?.permissions.write;
+
+  const { data: membersData, isLoading: isLoadingMembers } = useMemberships(slug);
+  if (isError) {
+    return <Http404 />;
+  }
+
+  const members = membersData?.filter((memberObject) => memberObject.membership_type === 'MEMBER').map((member) => member.user);
+  const membersSorted = members?.sort((a, b) => `${a.first_name} ${a.last_name}`.localeCompare(`${b.first_name} ${b.last_name}`));
+  return (
+    <Navigation banner={<Banner title={data?.name} />} fancyNavbar>
+      <Helmet>
+        <title>{data?.name}</title>
+      </Helmet>
+      {isLoadingGroups || isLoadingMembers ? (
```

Comment:
```suggestion
      {isLoadingGroups || isLoadingMembers || !data || !membersSorted ? (
```
Legg til sjekk her så slipper du optional chaining når du bruker dem lenger nede